### PR TITLE
MPAS Docs Update

### PR DESCRIPTION
**Pull request by phoban01**
## Description

Update the MPAS documentation site.

Signed-off-by: Piaras Hoban <phoban01@gmail.com>

### CODE REVIEW

**Review by morri-son — APPROVED** [review]
just added a link to the ocm.software page in the bootstrap.md

File: docs/site/src/boostrap.md [hunk 1]
```diff
@@ -1 +1,174 @@
...
+mpas bootstrap github \
+  --owner=<your-github-username> \
+  --repository=<your-github-repository> \
+  --path=clusters/my-cluster \
+  --dev \
+  --personal
+```
+
+If the specified repository does not exist, the `mpas bootstrap` command will create it
+as a private repository. If you wish to create a public repository, you can use the `--private=false`
```
Comment (Skarlso) [inline docs/site/src/boostrap.md:68]:
```suggestion
as a private repository. If you wish to create a public repository, you can use the `--private=false`
```

This seems a bit counterintuitive. Can we make it `--public` instead?

File: docs/site/src/getting_started.md [hunk 2]
```diff
@@ -0,0 +1,338 @@
...
+  -n mpas-system
+```
+
+2. Create a project that will contain the podinfo application.
+
+Let's create a directory for the project:
+
+```bash
+mkdir -p ./clusters/my-cluster/podinfo
+````
```
Comment (Skarlso) [inline docs/site/src/getting_started.md:128]:
```suggestion
```
```

Comment (Skarlso) [inline docs/site/src/getting_started.md:128]: There are 4 backticks here. I can't submit a suggestion to delete one. :D

File: docs/site/src/getting_started.md [hunk 3]
```diff
@@ -0,0 +1,338 @@
...
+Once the pull request is merged, `Flux` will detect the changes and deploy the application to the cluster.
+
+After a moment the `ProductDeployment` should be deployed successfully.
+It is possible to verify this with the command:
+
+```bash
+kubectl describe productdeployment -n mpas-podinfo-application  
+```
+
+The result should look something like:
```
[the remark this answers is not in the document]
Comment (phoban01) [reply docs/site/src/getting_started.md:319]: added a note

**Pull request merged**